browser-test/src/support/index.ts

```diff
@@ -205,3 +212,19 @@ export const validateAccessibility = async (page: Page) => {
 
   expect(results).toHaveNoA11yViolations()
 }
+
+export const validateScreenshot = async (
+  page: Page,
+  pageScreenshotOptions?: PageScreenshotOptions,
+  matchImageSnapshotOptions?: MatchImageSnapshotOptions,
+) => {
+  expect(
+    await page.screenshot({
+      ...pageScreenshotOptions,
+    }),
+  ).toMatchImageSnapshot({
+    failureThreshold: 0.03,
+    failureThresholdType: 'percent',
+    ...matchImageSnapshotOptions,
```

Comment:
Let's group screenshots by test directories matching test names. And skip "ts" in the direcory name. So for your test it will be be `__snapshots__/landing_page_test/the-landing-page-it-has-login-options-1-snap.png`
Though "snap" part also seems reduntant. Not sure what it supposed to indicate in their default schema.

Reply:
Done